spec/spec_helper.rb

```diff
@@ -1,6 +1,11 @@
 lib = File.expand_path('../lib', __FILE__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 
+
+require 'dor-workflow-service'
+require 'equivalent-xml'
+require 'equivalent-xml/rspec_matchers'
+
 Bundler.require(:default, :development)
```

Comment:
OK, but doesn't `Bundler.require` take care of all that?  Why doesn't it use the ':test' group?